Testing/Tests/graveyard_removal_test.py

```diff
@@ -0,0 +1,60 @@
+"""
+This class ensures that GraveyardRemoval.py correctly removes the graveyard from an h5m file.
+"""
+
+import os
+import pymoab
+import sys
+
+sys.path.insert(1,"../../PythonTool")
+
+from GraveyardRemoval import get_sets_by_category
```

Comment:
You can import multiple things with a comma separated list
```suggestion
from GraveyardRemoval import get_sets_by_category, locate_graveyard, format_file_name
```